tools/Dockerfile

```diff
@@ -36,6 +36,7 @@ RUN add-apt-repository ppa:rmescandon/yq && apt-get update -qq \
   groff \
   jq \
   yq \
+  awscli \
```

Comment:
Is this not included in the aws-authenticator then? I would like to keep our tools image lean and avoid cloud specific tooling as much as possible. We just removed all of that stuff.